NitroxServer/Communication/NitroxServer.cs

```diff
@@ -18,7 +18,7 @@ public abstract class NitroxServer
 
         protected readonly PacketHandler packetHandler;
         protected readonly EntitySimulation entitySimulation;
-        protected readonly Dictionary<int, NitroxConnection> connectionsByRemoteIdentifier = new();
+        protected readonly Dictionary<long, INitroxConnection> connectionsByRemoteIdentifier = new();
```

Comment:
Why do you change the identifier to a long?

Reply:
compatibility with programs that use a long as an identifier